doc/pages/user/015-astarte_dashboard.md

```diff
@@ -0,0 +1,394 @@
+
+- **Pairing** takes care of Device Authentication and Authorization. It interacts with Astarte's CA
+  and orchestrates the way devices connect and interact with Transports. It also handles Device
+  Registration. Agent, Device and Pairing interaction is described in detail
+  [here](050-pairing_mechanism.html).
+
+- **Flow** is the API endpoint for Astarte Flow, used for managing Blocks, Pipelines, and Flows.
+
+Possible statuses are:
+
+1. This service is operating normally.
+2. This service appears offline.
+
+A general status on API health is also present in the app's sidebar, thus always providing a
+feedback regardless of which page you are currently visiting.
+
+### Realm resources
+
+Within the main overview, a brief summary is available as well for the existing resources of the
+realm: registered and connected devices, installed interfaces, and installed triggers.
+
+More detailed overviews of each resource are available in the dedicated Dashboard sections,
+accessible via the navigation links in the Dashboard's side menu.
+
+## Interfaces
+
+[Interfaces](030-manage_interfaces.html) are a core concept of Astarte which defines how data is
+exchanged between Astarte and its peers.
+
+You can navigate to this section thanks to the Dashboard's side menu. A list of all installed
+interfaces is displayed, together with their major versions.
+
+![Astarte Dashboard, interface list](assets/astarte-dashboard-interfaces.png)
+
+### Installing interfaces
+
+From the Interface list, clicking on the **Install a new interface** button will load up the
+**Interface Editor**, an interactive tool that you can use to configure your interfaces.
+
+The Interface Editor provides you with two ways to define your interfaces: on the left panel, a
+graphical frontend, while on the right panel you may input a JSON definition to achieve the same
+result. Each panel updates automatically whenever the other is changed.
+
+![Astarte Dashboard, Interface Editor](assets/astarte-dashboard-interface-editor.png)
+
+While defining a new interface, the Interface Editor will help you in supplying the right options
+and filling in mandatory entries such as:
+
+- Name: an arbitrary name, formatted in reversed DNS casing.
+- Major and Minor versions: based on [Semantic Versioning](https://semver.org/).
+- Type: indicates whether data is streamed continuously (`datastream`) or is stateful and persistent
+  (`properties`).
+- Ownership: the write-only allowed actor. All the other actors are read-only.
+- Mappings: a list of endpoints that represent the data structure, following REST controller
+  semantics.
+
+You can learn more about Interface definitions in the
+[documentation's reference](030-interface.html).
+
+Note that when creating interface drafts, or for testing purposes in general, it is recommended to
+use `0` as the major version: to prevent data loss, Astarte allows only interfaces with
+`major_version` == `0` to be deleted.
```

Comment:
in a text description i'd avoid equal sign to express "equals"

Reply:
I agree, fixed it.